components/why-koda/HeroSection.vue

```diff
@@ -0,0 +1,126 @@
+<template>
+  <section class="border-b relative max-md:flex max-md:flex-col-reverse">
+    <div class="container">
+      <div class="max-md:px-4 max-md:py-16 md:py-[96px]">
+        <h1 class="flex items-center gap-6 flex-wrap items-center">
+          <span class="text-6xl font-bold text-center capitalize md:inline">
+            {{ $t('why') }}
+          </span>
+          <div class="h-[calc(100%-13px)] flex items-center">
+            <img :src="logoSrc" alt="koda-logo" class="h-full" />
+          </div>
+        </h1>
+
+        <p class="!mt-6 text-xl">
+          {{ $t('whyKoda.embraceTheFutureOfGenerativeArt') }}
+        </p>
+
+        <div class="!mt-10 md:mt-16">
+          <ol class="text-xl md:text-2xl !pl-7 list-decimal w-[300px]">
+            <li
+              v-for="section in sections"
+              :key="section.id"
+              class="!mb-3 border rounded-full px-2 md:!px-3">
+              <NeoButton
+                v-safe-href="`#${section.id}`"
+                variant="text"
+                tag="a"
+                no-shadow
+                class="text-xl md:text-2xl">
+                {{ section.name }}
+              </NeoButton>
+            </li>
+          </ol>
+        </div>
+      </div>
+    </div>
+
+    <div
+      class="md:absolute top-0 right-0 md:w-[55%] h-full overflow-hidden bg-neutral-3 dark:bg-neutral-11 max-md:border-b md:border-l">
+      <div
+        class="max-md:py-9 flex flex-col max-md:gap-4 md:gap-8 h-full justify-center">
+        <div
+          v-for="(nfts, index) in nftsByStrip"
+          :key="`strip-${index}`"
+          class="flex max-md:gap-4 md:gap-8"
+          :class="{
+            'transform -translate-x-60': index !== 1,
+            'transform translate-x-4 md:translate-x-8': index === 1,
```

Comment:
maybe `first` or `last` can be your friend here?
https://tailwindcss.com/docs/hover-focus-and-other-states#first-last-odd-and-even

Reply:
thanks!, fixed